src/runtime_src/core/edge/user/aie/graph.cpp

```diff
@@ -1037,3 +1037,250 @@ xclStopProfiling(xclDeviceHandle handle, int phdl)
   }
```

Comment:
Copyright needs to be updated (line 3).  Being that we are part of AMD now, the following addition should be made.
`Copyright (C) Advanced Micro Devices, Inc.`